packages/cli/src/services/user.service.ts

```diff
@@ -65,11 +65,11 @@ export class UserService {
 	) {
 		const { password, updatedAt, authIdentities, mfaRecoveryCodes, mfaSecret, ...rest } = user;
 
-		const ldapIdentity = authIdentities?.find((i) => i.providerType === 'ldap');
+		const providerType = authIdentities?.[0]?.providerType;
```

Comment:
Isn't it possible that the user has 2 auth identities ? or do we admit that there can be only one (in that case this is not future proof, but fine i guess)